app/helpers/effective_datatables_helper.rb

```diff
@@ -50,14 +49,15 @@ def render_datatable(datatable, input_js: {}, buttons: true, charts: true, entri
         'display-records' => datatable.to_json[:recordsFiltered],
         'display-start' => datatable.display_start,
         'inline' => inline.to_s,
+        'language' => EffectiveDatatables.language(I18n.locale),
```

Comment:
Passed in from the view, so this should still be all dynamic and per user if needed